forte/core.py

```diff
@@ -66,3 +69,40 @@ def __new__(cls):
 
     def __del__(cls):
         forte.cleanup()
+
+
+def start_logging():
+    """This function starts logging"""
+    logging.basicConfig(filename='forte.log', level=logging.DEBUG, format='# %(asctime)s | %(levelname)s | %(message)s')
+    logging.info('Starting the Forte logger')
+
+
+def flog(level, msg):
+    """
+    Log the message ``msg`` with logging level ``level``.
+
+    This function calls the logging module and puts some spaces before
+    the message that we want to log.
+    """
+    global logging_depth
+    if level == 'info':
+        spaces = ' ' * max((logging_depth - 1), 0) * 2
+        logging.info(f"{spaces}{msg}")
+
```

Comment:
Raise an error rather than fail silently, if no logging level is set.

Reply:
I implemented the other logging levels and added an exception if fails.